_rules/zoom-text-no-overflow-clipping-59br37.md

```diff
@@ -0,0 +1,247 @@
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This [text node][] is fully [visible][] at a [viewport size][] of 640 by 512.
+
+```html
+<div style="white-space: nowrap; overflow: hidden;">
+	Once upon a midnight dreary, while I pondered, weak and weary,<br />
+	Over many a quaint and curious volume of forgotten lore.<br />
+	While I nodded, nearly napping, suddenly there came a tapping,<br />
+	As of some one gently rapping, rapping at my chamber door.<br />
+	“’Tis some visitor,” I muttered, “tapping at my chamber door.<br />
+	Only this and nothing more.”
+</div>
+```
+
+#### Passed Example 2
+
+This [text node][] is [clipped][] using `text-overflow: ellipsis` at a [viewport size][] of 640 by 512. A link to a full version of the poem is also provided.
+
+```html
+<div style="white-space: nowrap; overflow: hidden; text-overflow: ellipsis;">
+	Once upon a midnight dreary, while I pondered, weak and weary, Over many a quaint and curious volume of forgotten
+	lore. While I nodded, nearly napping, suddenly there came a tapping.
+</div>
+<a href="/test-assets/59br37/poem.html">Full text of the poem</a>
+```
+
+#### Passed Example 3
+
+This [text node][] is restricted to a single line, by setting a `line-height` that is the same as the `height`. A link to a full version of the poem is also provided.
+
+```html
+<style>
+	.wordClip {
+		overflow: hidden;
+		word-wrap: break-word;
+		height: 16px;
+		line-height: 16px;
+	}
+</style>
+<div class="wordClip">
+	Once upon a midnight dreary, while I pondered, weak and weary, Over many a quaint and curious volume of forgotten
+	lore. While I nodded, nearly napping, suddenly there came a tapping.
+</div>
+<a href="/test-assets/59br37/poem.html">Full text of the poem</a>
+```
+
+#### Passed Example 4
+
+This [text node][] is not [clipped][] with `overflow: hidden` because it has a parent with `overflow: auto` at a [viewport size][] of 640 by 512.
+
+```html
+<div style="overflow: hidden; height: 2em;">
+	<div style="overflow: auto; height: 2em;">
+		Once upon a midnight dreary, while I pondered, weak and weary, Over many a quaint and curious volume of forgotten
+		lore. While I nodded, nearly napping, suddenly there came a tapping.
+	</div>
+</div>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This [text node][] is [clipped][] because it has a fixed height that does not leave enough space for the content to wrap.
+
+```html
+<div style="overflow: hidden; height: 1.5em;">
+	Once upon a midnight dreary, while I pondered, weak and weary, Over many a quaint and curious volume of forgotten
+	lore. While I nodded, nearly napping, suddenly there came a tapping.
+</div>
+```
+
+#### Failed Example 2
+
+This [text node][] is [clipped][] because its height is relative to the viewport height, leaving insufficient space for the page to be zoomed to 200% at a [viewport size][] of 640 by 512.
```

Comment:
```suggestion
This [text node][] is [clipped][] because its height is relative to the viewport height, leaving insufficient space for the page to be zoomed to 200% at a [viewport size][] of 1280 by 1024.
```